src/client/containers/ChannelInformationContainer/ChannelInfo.js

```diff
@@ -11,13 +11,22 @@ export default function ChannelInfoRender() {
   useEffect(() => {
     const fetchChannelAndMembers = async () => {
       const channel = await fetchWithAuth(`/api/channels/${id}`);
-      setchannelTitle(channel[0].title);
-      if (channel[0].url) {
-        setchannelImageUrl(channel[0].url);
+      const membersofChannel = await fetchWithAuth(
```

Comment:
You could simplify all of this if the `GET /api/channel/:channelId` endpoint returns these when called.